### PR TITLE
[multimodal] Refine detection evaluation output

### PR DESCRIPTION
**Pull request by FANGAreNotGnu**
Add constants for detection evaluation metrics and parsing the output for pycocotools output. Naming follows torchmetric.

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

File: multimodal/src/autogluon/multimodal/constants.py [hunk 1]
```diff
@@ -64,8 +64,21 @@
 SCORE = "score"
 LOGIT_SCALE = "logit_scale"
 
-# Metric
+# Metric for Object Detection
 MAP = "map"
```
Comment (bryanyzhu) [inline multimodal/src/autogluon/multimodal/constants.py:68]: One question, MAP standards for mean average precision, will it be confusing to other people when they see MEAN_AVERAGE_PRECISION in line 82?

Comment (FANGAreNotGnu) [reply multimodal/src/autogluon/multimodal/constants.py:68]: Good point, edited the code to make MEAN_AVERAGE_PRECISION also work (=MAP).

File: multimodal/src/autogluon/multimodal/constants.py [hunk 2]
```diff
@@ -64,9 +64,37 @@
...
+MAP_SMALL = "map_small"
+MAP_MEDIUM = "map_medium"
+MAP_LARGE = "map_large"
+MAR_1 = "mar_1"
+MAR_10 = "mar_10"
+MAR_100 = "mar_100"
+MAR_SMALL = "mar_small"
+MAR_MEDIUM = "mar_medium"
+MAR_LARGE = "mar_large"
+DETECTION_METRICS = [
```
Comment (zhiqiangdon) [inline multimodal/src/autogluon/multimodal/constants.py:81]: Do we need tests for object detection metrics?

**Review by zhiqiangdon — APPROVED** [review]
LGTM!

File: multimodal/src/autogluon/multimodal/constants.py (hunk 2 above)
Comment (FANGAreNotGnu) [reply multimodal/src/autogluon/multimodal/constants.py:81]: Oh sorry for noticing this after the merge. I'll add the test to it in an existing one in next pr.

**Pull request merged**